src/features/profile/index.tsx

```diff
@@ -0,0 +1,355 @@
+          <div className={styles.row}>
+            <div className={styles.row_title}>Total sales amount</div>
+            <div className={styles.row_content}>560,600,00k PSL</div>
+          </div>
+          <div className={styles.row}>
+            <div className={styles.row_title}>Total items sold</div>
+            <div className={styles.row_content}>14</div>
+          </div>
+          <div className={styles.row}>
+            <div className={styles.row_title}>Top category persentage</div>
+            <div className={styles.row_content}>motion graphics 30%</div>
+          </div>
+        </div>
+        {/* Third Row Group of General */}
+        <div className={classnames('w-full py-8')}>
+          <div className={styles.row}>
+            <div className={styles.row_title}>Bio</div>
+          </div>
+          <div className={styles.row}>
+            <div className={styles.row_content} style={{ fontSize: 14 }}>
+              I'am baby readymade mikshk tatooed actually activated charcoal
+              godard listicle. Mumblecore cronut kickstarter, bushwick wolf
+              copper mug woke chia put a bird on it viral gentrify keytar synth.
+              Twee chartreuse etsy, +1 dreamcatcher lumbersexual before they
+              sold out drinking vinegar pinterest mumblecore tousled occupy
+              brunch whatever ugh.
+            </div>
+          </div>
+        </div>
+      </div>
+    )
+  }
+}
+
+class Relations extends Component<any, any> {
+  render() {
+    const {} = this.props
+    return (
+      <div className='w-full xl:w-1/2 flex flex-col flex-grow px-4'>
+        <div className='flex'>
+          <div className={classnames(styles.tab, styles.active)}>
+            Followers (235)
+          </div>
+          <div className={classnames(styles.tab)}>Following (162)</div>
+          <div className={classnames(styles.tab)}>Mutual (73)</div>
+        </div>
+        <Followers followers={followers} />
+      </div>
+    )
+  }
+}
+
+class Followers extends Component<any, any> {
+  render() {
+    const { followers } = this.props
+    return (
+      <div className='flex flex-col pt-2 pr-4'>
+        {followers.map((follower: any, index: any) => (
+          <Follower follower={follower} key={index} />
+        ))}
+      </div>
+    )
+  }
+}
+
+class Follower extends Component<any, any> {
+  render() {
+    const { follower } = this.props
+    return (
+      <div className='flex items-center py-2 text-md pr-8'>
+        <div
+          className={classnames(
+            'rounded-full bg-pink-300 w-10 h-10',
+            styles.follower_avatar,
+          )}
+        ></div>
+        <div
+          className={classnames(
+            'flex-grow font-bold pl-4',
+            styles.follower_name,
+          )}
+        >
+          {' '}
+          {follower.name}{' '}
+        </div>
+        <div className={classnames('text-gray-400 text-sm')}>161 followers</div>
+      </div>
+    )
+  }
+}
```

Comment:
it has a lot of functions and components in this file. Can we create new components? Please refer here: 

https://github.com/pastelnetwork/pastel-electron-wallet/tree/master/src/features/receive